packages/sdk-angular/projects/fusionauth-angular-sdk/src/lib/fusion-auth.service.spec.ts

```diff
@@ -126,19 +126,4 @@ describe('FusionAuthService', () => {
 
     expect(onRedirect).toHaveBeenCalledWith('/welcome-page');
   });
-
-  it("Invokes an 'onAutoRefreshFailure' callback", fakeAsync(() => {
-    mockIsLoggedIn();
-    spyOn(window, 'fetch').and.resolveTo(new Response(null, { status: 400 }));
-    const onAutoRefreshFailure = jasmine.createSpy('onAutoRefreshFailureSpy');
-
-    configureTestingModule({
-      ...config,
-      shouldAutoRefresh: true,
-      onAutoRefreshFailure,
-    });
-
-    tick(60 * 60 * 1000);
-    expect(onAutoRefreshFailure).not.toHaveBeenCalledWith();
```

Comment:
I realized that this test has an incorrect assertion. Based on the setup above, it _should_ expect `onAutoRefreshFailure` to have been called. After troubleshooting for multiple hours, I decided to remove the test. I manually tested with the quickstart to confirm that the callback is invoked on failure. Jasmine's time controlling API isn't flexible enough to work for our use case. I'm removing it because I'd rather have no test, than one with an assertion that doesn't match the description.